2018/venue.dd

```diff
@@ -1,10 +1,11 @@
 Ddoc
 $(T h1,, Venue)
 $(P
-Like to hit the snooze button a few times before a busy day at DConf? This year, you can get a couple extra snoozes in than you could at past iterations! DConf 2018 is taking place at the NH München Messe, a 253-room hotel in the Zamdorf area of Munich.
+Like to hit the snooze button a few times before a busy day at DConf? This year, you'll be able to hit it a couple of times more than you could at past editions! DConf 2018 is taking place at the NH München Messe, a 253-room hotel in the Zamdorf area of Munich.
 )
+$(P <img src="images/NH_hotel.jpg">)
 $(P
-Not only will this save you time, it will save you money. The hotel is offering a discount to DConf 2018 attendees. Keep an eye on this space for information on how to take advantage of this offer.
+Not only will this save you time, it will save you money. The hotel is offering a discount to DConf 2018 attendees. To take advantage of this offer and book a single room (including breakfast) for &#8364;89/night, send an email to $(LINK2 mailto:&#114;&#101;&#115;&#101;&#114;&#118;&#105;&#101;&#114;&#117;&#110;&#103;&#101;&#110;&#64;&#110;&#104;&#45;&#104;&#111;&#116;&#101;&#108;&#115;&#46;&#99;&#111;&#109;, &#114;&#101;&#115;&#101;&#114;&#118;&#105;&#101;&#114;&#117;&#110;&#103;&#101;&#110;&#64;&#110;&#104;&#45;&#104;&#111;&#116;&#101;&#108;&#115;&#46;&#99;&#111;&#109;) with the keyword $(B D-Conf) in the subject line. Visit $(LINK2 https://www.nh-hotels.com/hotel/nh-muenchen-messe/map?_ga=2.24549178.1852131175.1517217246-33887194.1489576485, nh-hotels.com) for directions and points of interest.
```

Comment:
`&euro;` is typically a lot easier.

Fun fact: I am aware that €89 is the correct English typography, but I'm not sure whether you are aware in German it would be 89€.

Reply:
I wasn't aware of the typography. And I was going to say that I tried &euro; first and Ddoc passed it on through, but looking back through my editor history I see I typed it without the semi-colon.